internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -850,6 +850,19 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/Filesystem'
+        services:
+          type: object
+          properties:
+            enabled:
+              type: array
+              items:
+                type: string
+                example: "nftables"
+            disabled:
```

Comment:
Same here, a `description` and `minItems` would be nice I think.

Reply:
Done